src/assets/scss/app.scss

```diff
@@ -241,7 +241,7 @@ $btn-focus-box-shadow: 0 0 0 3px rgba(0, 0, 0, 0.18);
   &:hover, &:focus, &:active:focus {
     background-color: $color-danger-dark;
 
-    &[aria-disabled="true"], &.disabled, fieldset[disabled] {
+    &[aria-disabled="true"], &.disabled, fieldset[disabled] & {
```

Comment:
This isn't really related to this PR, I just noticed that this style was incorrect and was missing an `&`. It doesn't follow the same pattern as other buttons.